README.md

```diff
@@ -1 +1,92 @@
-# angular-resource-readme
\ No newline at end of file
+# $resource
+
+## Overview
+
+We've had a look at the awesome `$http` service that Angular provides, but there is also another awesome service under the name of `$resource`. Let's take a look at that!
+
+## Objectives
+
+- Describe $resource
+- Use $resource methods
+- Fetch data from an API
+- Update data from an API
+
+## What is $resource?
+
+$resource is a factory that creates a resource object for us to communicate with APIs. We pass through a URL and it gives us an object back that looks like this:
+
+```js
+var User = $resource('/user/:userId');
+
+/**
+ * User = { get: function (),
+ *          save: function (),
+ *          remove: function ()
+ *        }
+ */
+```
+
+Here, we've defined our URL. Let's break that down:
+
+`/user/:userId`
+
+We're defining our URL as `/user/:userId` - but hold on, we can't have colons in our URLs!?
+
+Well, with $resource, using a colon in a URL means that section is *actually* a variable. `:userId` means that we aren't actually going to put `:userId` into our URL, rather that we will be replacing `:userId` with an actual user ID. This allows us to put our data into our URL - for instance, we would end up querying `/user/1` or `/user/1231938`.
```

Comment:
This makes sense - can you add a little bit more detail about what the `$resource` is inferring from that URL - how does it know where to make the proper get, post, and delete requests?